src/Containers/ContributionForm/ContributionForm.js

```diff
@@ -80,7 +111,7 @@ function ContributionForm() {
   }
 
   const makeList = () => {
-    if(specialties) {
+    if(specialties.length) {
```

Comment:
i'm curious what the need for this is?

Reply:
specialties is an array, and so an empty array is still truthy, so that's why I changed it

Reply:
oh, good catch!